src/SiteExtensions/dependencies.props

```diff
@@ -0,0 +1,11 @@
+﻿<Project>
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>2.1.1-rtm-31076</MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>
+    <MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion></MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>
```

Comment:
This is empty on purpose?

Reply:
Waiting for a package to be produced from 2.2 branch.

Reply:
Got it!